charts/nr-k8s-otel-collector/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for nr-k8s-otel-collector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+kube-state-metrics:
+  # kube-state-metrics.enabled -- Install the [`kube-state-metrics` chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics) from the stable helm charts repository.
+  # This is mandatory if `infrastructure.enabled` is set to `true` and the user does not provide its own instance of KSM version >=1.8 and <=2.0. Note, kube-state-metrics v2+ disables labels/annotations
+  # metrics by default. You can enable the target labels/annotations metrics to be monitored by using the metricLabelsAllowlist/metricAnnotationsAllowList options described [here](https://github.com/prometheus-community/helm-charts/blob/159cd8e4fb89b8b107dcc100287504bb91bf30e0/charts/kube-state-metrics/values.yaml#L274) in
+  # your Kubernetes clusters.
+  enabled: true
+
+image:
+  repository: otel/opentelemetry-collector-contrib
```

Comment:
should we use nrdot?

Reply:
Can you clarify what you mean by using nrdot? I'm assuming this means nrdot has a distribution of otel that theyre using, and if thats the case i'd have to confirm that theyre including all of the processors/receivers/exporters that we're using, but its probably not strictly necessary for what we're trying to achieve at this time w/ this pr.

Reply:
I mean [this](https://github.com/newrelic/opentelemetry-collector-releases/tree/main/distributions/nr-otel-collector) I think @mangulonr can take a final decision on that.
But for sure receivers are missing

Reply:
We need to provide NRDOT as the default option in the Helm Chart.
NRDOT components are in this [manifest](https://github.com/newrelic/opentelemetry-collector-releases/blob/main/distributions/nr-otel-collector/manifest.yaml), I assume we need what is needed to properly monitor a K8s cluster.

As part of our documentation, we will document how to use a 3rd party OTel collector if required by our customers (including config and components needed).

@csongnr let's sync in our today's meeting